content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst

```diff
@@ -117,14 +113,46 @@ serves as an additional tool to update the unit price of products.
    :alt: Add manual valuation of stock value to a product.
 
 Create manual valuation entries on the :guilabel:`Stock Valuation` dashboard by first navigating to
-:menuselection:`Inventory app --> Reporting --> Inventory Valuation`, and then click the little gray
-drop-down icon to the left of a product's name, to then reveal stock valuation line items below as
-well as a teal :guilabel:`+ (plus)` button on the right.
+:menuselection:`Inventory app --> Reporting --> Valuation`. Next, to enable the product revaluation
```

Comment:
Italicizing and capitalizing the *Product Revaluation* button because we are referring to a UI element outside of the active instruction
```suggestion
:menuselection:`Inventory app --> Reporting --> Valuation`. Next, to enable the *Product Revaluation*
```